lib/twitter/twits.ex

```diff
@@ -0,0 +1,145 @@
+defmodule Twitter.Twits do
+  @moduledoc """
+  The Twits context.
+  Includes functions for posts and likes
+  """
+
+  import Ecto.Query, warn: false
+  alias Twitter.Repo
+  alias Twitter.Twits.Post
+  alias Twitter.Twits.Like
+  alias Twitter.Accounts
+
+  defp user_posts_query(query, %Accounts.User{id: user_id}) do
+    from p in query, where: p.user_id == ^user_id
+  end
+
+  @doc """
+  Returns the list of posts. This fn is used for the current user.
```

Comment:
It is best to avoid abbreviations to increase readability.
```suggestion
  Returns the list of posts. This function is used for the current user.
```
In this case there is little doubt of what you mean, but it is still a bit misleading since `fn` in elxir means anonymous function, and this one is not.